src/BookDetails/BookDetails.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+
+export const BookDetails = ({book}) => {
+
+    return(
+        <section>
+          <button>Close Details</button>
+          <article>
+            <img src={book.image}/>
+            <h2>{book.title}</h2>
+            <p>{book.summary}</p>
+          </article>
+          <article>
+            <div>PROGRESS BAR</div>
+            <button>Add A Bookmark</button>
```

Comment:
this button needs to open new bookmark form (via conditional rendering or router??)